suite-native/module-home/src/screens/HomeScreen/HomeScreen.tsx

```diff
@@ -39,12 +39,13 @@ export const HomeScreen = () => {
 
     return (
         <Screen
-            screenHeader={<DeviceManagerScreenHeader hasBottomPadding />}
+            screenHeader={<DeviceManagerScreenHeader />}
             refreshControl={refreshControl}
             customHorizontalPadding={0}
+            customTopPadding={0}
```

Comment:
One question... Why is the customTopPadding needed everywhere? Could not it be set as a default value of the `Screen` component? Like that we would not have to pass it everywhere manually in so many places.

Reply:
Because the default is 16px, not zero. But there are some use cases where zero is needed. I'd reduce it to a boolean flag in the future if possible, we'll be looking into it again with @shenkys.